templates/contribute/index.md

```diff
@@ -10,10 +10,23 @@ to make the process of contributing as smooth as possible.
    - The explanation of [naming conventions](naming.html).
    - The [documentation guidelines](doc.html).
 
-Once you have code that you'd like to contribute, you should open a PR.
+## What to contribute to mathlib
```

Comment:
Somewhere you should also say: "Discuss on Zulip whether this is a good topic to add"